src/BlazorWebView/src/Maui/Android/AndroidWebKitWebViewManager.cs

```diff
@@ -62,20 +64,33 @@ protected override void SendMessage(string message)
 
 		internal void SetUpMessageChannel()
 		{
-			var nativeToJsPorts = _webview.CreateWebMessageChannel();
+			_nativeToJSPorts = _webview.CreateWebMessageChannel();
 
 			var nativeToJs = new BlazorWebMessageCallback(message =>
 			{
 				MessageReceived(new Uri(AppOrigin), message!);
 			});
 
-			var destPort = new[] { nativeToJsPorts[1] };
+			var destPort = new[] { _nativeToJSPorts[1] };
 
-			nativeToJsPorts[0].SetWebMessageCallback(nativeToJs);
+			_nativeToJSPorts[0].SetWebMessageCallback(nativeToJs);
 
 			_webview.PostWebMessage(new WebMessage("capturePort", destPort), AndroidAppOriginUri);
 		}
 
+		public async new ValueTask DisposeAsync()
```

Comment:
I think you want to just override this virtual method from the base class: https://github.com/dotnet/aspnetcore/blob/main/src/Components/WebView/WebView/src/WebViewManager.cs#L223

Reply:
Ah good catch, thanks! ffeef2536